### ISSUE TITLE
Grid View makes participants lined up vertically

### ISSUE THREAD

**Issue by SalamatShopee1** [Fugiman/google-meet-grid-view#345]
Windows 10 Update 20H2
Google Chrome
![2021-02-18 09_53_32-](https://user-images.githubusercontent.com/79236093/108294141-3c13d100-71d0-11eb-8ca1-177a4e838766.png)

**Comment by Jkarhani** [comment]
Yes, it seems that google stopped it by updating itself..we should ask google to enable it back.. Please let us all complain.

**Comment by fedefader** [comment]
it's my sames problem, also i have this problem!!!

**Comment by Eagle-Man** [comment]
From the Grid View (Fix) GitHub page:

> UPDATE 02/11/2021:
> It turns out that Google is testing an update of Meet that disables the Grid View extension. This is why sometimes it works and sometimes not (you see the boxes in a vertical bar). If you seriously need Grid View right now, you might consider refreshing a few times the browser tab (you should end up loading the old compatible version at a certain point).
> I am really sorry but I have to temporarily remove the extension from Chrome Web Store. In fact, updating Grid View would require a huge amount of work or may not be feasible at all. Please, remember that Google Meet introduced native grid view (up to 49 tiles).

**Comment by Eagle-Man** [comment]
@Jkarhani The solution is NOT to ask Google to "put it back."  There's no way they're going to customize their code for one third-party extension.  The real solution is to tell Google what would make Meet better for you; what features you want to see in Meet.  And if enough people do this, maybe we'll see some of them native to Meet where they can't be broken again.

**Comment by fedefader** [comment]
I see!  I thank you for the answer!  I will contact Google

Federico F.
